src/widgets/SearchBar.js

```diff
@@ -132,7 +133,7 @@ SearchBarComponent.defaultProps = {
   debounceTimeout: 250,
   textInputStyle: defaultStyles.textInput,
   viewStyle: defaultStyles.container,
-  color: SUSSOL_ORANGE,
+  color: WHITE,
```

Comment:
again we've changed the default prop back to `WHITE` .. we only wanted to change the colour of the component to white when it is used in `AutoCompleteSelector`

Reply:
This one is a mistake and confusion.

Reply:
As Josh pointed out, you have changed the default prop to `WHITE` rather than the usually desired `SUSSOL_ORANGE`. While this does look correct in the AutoCompleteSelector:
![Screen Shot 2019-12-16 at 12 26 15](https://user-images.githubusercontent.com/7684221/70871027-9da24380-1fff-11ea-9832-8494ebf47d3b.png)

It breaks the design in every other table page in the app which also use the `SearchBar` component:
![Screen Shot 2019-12-16 at 12 27 57](https://user-images.githubusercontent.com/7684221/70871033-bad71200-1fff-11ea-8176-9515cade973d.png)

Reply:
That said, this file doesn't need any changes.

Reply:
Sorry, this was a mistake..